src/scales/scale.time.js

```diff
@@ -10,6 +10,10 @@ module.exports = function(Chart) {
 
 	var timeHelpers = helpers.time;
```

Comment:
Yeah totally agree, that's my next task with some optimizations if this PR get merged